[BUG] Can't resolve '@/node_modules/next-auth/providers/email'

When running locally I'm seeing this error

```
./utils/app/auth/providers.ts:97:27
Module not found: Can't resolve '@/node_modules/next-auth/providers/email'
   95 |   }
   96 |   if (EMAIL_FROM) {
>  97 |     const provider = await import('@/node_modules/next-auth/providers/email');
      |                           ^
   98 |     const EmailProvider = provider.default;
   99 |     providers.push(
  100 |       EmailProvider({

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./app/api/auth/[...nextauth]/route.ts

```

--- a/docs/docs/model-providers/openai.md
+++ b/docs/docs/model-providers/openai.md
@@ -42,6 +42,11 @@ The base url for the API.
 OPENAI_API_URL="https://api.openai.com/v1"
 ```
 
+It is possible to use the new [Cloudflare AI Gateway](https://developers.cloudflare.com/ai-gateway/)
+```sh
+OPENAI_API_URL="https://gateway.ai.cloudflare.com/v1/ACCOUNT_TAG/GATEWAY"
+```
+
 ---
 
 ### OPENAI_API_TYPE
@@ -81,7 +86,7 @@ The default system prompt for OpenAI models.
 #### Example
 
 ```sh
-NEXT_PUBLIC_DEFAULT_OPENAI_SYSTEM_PROMPT= "You are ChatGPT, a large language model trained by OpenAI. Follow the user's instructions carefully. Respond using markdown."
+NEXT_PUBLIC_DEFAULT_OPENAI_SYSTEM_PROMPT="You are ChatGPT, a large language model trained by OpenAI. Follow the user's instructions carefully. Respond using markdown."
 ```
 
 ---

--- a/packages/unsaged/utils/app/auth/providers.ts
+++ b/packages/unsaged/utils/app/auth/providers.ts
@@ -94,7 +94,7 @@ export async function getProviders() {
     );
   }
   if (EMAIL_FROM) {
-    const provider = await import('@/node_modules/next-auth/providers/email');
+    const provider = await import('next-auth/providers/email');
     const EmailProvider = provider.default;
     providers.push(
       EmailProvider({

